openshift_scalability/content/statefulset-pv-template.json

```diff
@@ -51,6 +51,16 @@
               {
                 "name": "server",
                 "image": "${IMAGE}",
+                "resources": {
+                  "requests": {
+                    "memory": "128Mi",
```

Comment:
Thanks for the comment.
Let me make those as parameters.

To answer your question, the flow we use this template for the moment is:
cluster-loader will do "oc process" and then "oc create" on this template.